src/components/Fundraiser.tsx

```diff
@@ -29,18 +29,28 @@ interface FundraiserState extends JSONObject {
   goal: number | null;
   nonprofitInfo: EveryNonprofitInfo | null;
   supporters: number;
+  status: FundraiserStatus | 'unknown';
```

Comment:
'unknown' should be part of FundraiserStatus or null. Setting a magic string here and other places will cause issues down the line